Library/Homebrew/version.rb

```diff
@@ -415,7 +415,7 @@ def self._parse(spec, detected_from_url:)
     # e.g. foobar-4.5.1-1
     # e.g. unrtf_0.20.4-1
     # e.g. ruby-1.9.1-p243
-    StemParser.new(/[_-](#{NUMERIC_WITH_DOTS}-(?:p|rc|RC)?\d+)#{CONTENT_SUFFIX}?$/),
+    StemParser.new(/[_-](#{NUMERIC_WITH_DOTS}-(?:p|P|rc|RC)?\d+)#{CONTENT_SUFFIX}?$/),
```

Comment:
Could this whole regex perhaps just be made case-insensitive?

Reply:
Probably, but I didn't in this case because none of the other regexes in the list had any flags tacked onto the end.

Reply:
I'm fine with it either way.